db/view/view.cc

```diff
@@ -1213,6 +1213,25 @@ view_builder::view_builder(database& db, db::system_distributed_keyspace& sys_di
         : _db(db)
         , _sys_dist_ks(sys_dist_ks)
         , _mnotifier(mn) {
+    setup_metrics();
+}
+
+void view_builder::setup_metrics() {
+    namespace sm = seastar::metrics;
+
+    _metrics.add_group("view_builder", {
```

Comment:
Please use the method form that the function/param is the last parameters

Reply:
I would gladly, but `sm::make_queue_length` does not have this form at all, and I'd prefer to use `make_queue_length` insted of the generic `gauge`, because it's more descriptive.

Reply:
... and I added the rest in the same form in order not to mix forms in a single call.